test/SimpleVersion.Core.Tests/Pipeline/Formatting/Semver1FormatProcessFixture.cs

```diff
@@ -17,6 +17,15 @@ public Semver1FormatProcessFixture()
         }
 
         public static IEnumerable<object[]> LabelParts()
+        {
```

Comment:
Once we have configuration options available to enable/disable the feature we'll need to move these separate test cases.